runatlantis.io/docs/apply-requirements.md

```diff
@@ -114,6 +114,10 @@ the Atlantis user needs to be part of that list in order for it to consider
 a pull request mergeable.
 :::
 
+::: warning
+If you set `atlantis/apply` to the mergeable requirement, use `--gh-allow-mergeable-bypass-apply` flag. This flag allows the mergeable check before `apply` to skip checking the status of `atlantis/apply`.
```

Comment:
Looks like the flag is documented here https://www.runatlantis.io/docs/server-configuration.html#gh-allow-mergeable-bypass-apply but it doesnt show the env var.

It also shows a bullet where there shouldn't be a bullet.

https://github.com/runatlantis/atlantis/blame/master/runatlantis.io/docs/server-configuration.md#L372

Reply:
@nitrocode Thanks for your comment!

I have fixed a wrong bullet and documented the env var to `apply-requirements.md`. On the other hand, I have not appended the env var to `server-configuration.md` because there is no mention of env vars in the other flags (Excluding a few. Lack uniformity...).

Reply:
Ah I had a small typo https://github.com/runatlantis/atlantis/pull/2583